W-14952310: x-ratelimit header warning (#520)
mulesoft/docs-gateway: gateway/1.5/modules/ROOT/pages/policies-included-rate-limiting-sla.adoc

By default, the X-RateLimit headers are disabled in the response. You can enable these headers by selecting *Expose Headers* when you configure the policy.

[NOTE]
====

> When on distributed mode the X-Ratelimit-Remaining header is not kept in perfect synchronization with all the replicas.  With multiple replicas on distributed mode the X-Ratelimit-Remaining header can be thought of as an estimation of how much quota te replica has available for itself and not the exact value of the remaining actual quota available for the API.

Reviewer: ```suggestion
When on distributed mode, the X-Ratelimit-Remaining header is not perfectly synchronized with all the replicas.  With multiple replicas on distributed mode, the X-Ratelimit-Remaining header is an estimation of how much quota the replica has available for itself. It does not represent the actual value of the remaining quota available for the API.
```

> To illustrate this here is a valid scenario:

Reviewer: ```suggestion
The following describes a valid scenario:
```

* Created a SLA tier name "Titanium" having rate limit of "999999999 req per 10 min"
* Sent 1 request to replica 1 -> x-ratelimit-remaining : 999999998
* Sent 1 request to replica 2 -> x-ratelimit-remaining : 899999998

> This does not mean that the quota will be surpassed.

Reviewer: ```suggestion
In this case the quota will not be surpassed.
```
